docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-c.md

```diff
@@ -0,0 +1,192 @@
+menu:
+  latest:
+    parent: cloud-build-apps
+    name: C
+    identifier: cloud-c
+    weight: 500
+type: page
+isTocNested: true
+showAsideToc: true
+---
+
+The following tutorial shows a small [C application](https://github.com/yugabyte/yugabyte-simple-c-app) that connects to a YugabyteDB cluster using the [libpq driver](../../../../reference/drivers/ysql-client-drivers/#libpq) and performs basic SQL operations. Use the application as a template to get started with Yugabyte Cloud in C.
+
+## Prerequisites
+
+- 32-bit (x86) or 64-bit (x64) architecture machine. (Use [Rosetta](https://support.apple.com/en-us/HT211861) to build and run on Apple silicon.)
+- gcc 4.1.2 or later, or clang 3.4 or later installed.
+- OpenSSL 1.1.1 or later (used by libpq to establish secure SSL connections).
+- [libpq](../../../../reference/drivers/ysql-client-drivers/#libpq). Homebrew users on macOS can install using `brew install libpq`. You can download the PostgreSQL binaries and source from the [PostgreSQL Downloads](https://www.postgresql.org/download/) page.
+
+### Yugabyte Cloud
+
+- You have a cluster deployed in Yugabyte Cloud. To get started, use the [Quick start](../../).
+- You downloaded the cluster CA certificate. Refer to [Download your cluster certificate](../../../cloud-secure-clusters/cloud-authentication/#download-your-cluster-certificate).
+- You have added your computer to the cluster IP allow list. Refer to [Assign IP Allow Lists](../../../cloud-secure-clusters/add-connections/).
+
+## Clone the application from GitHub
+
+Clone the sample application to your computer:
+
+```sh
+git clone https://github.com/yugabyte/yugabyte-simple-c-app && cd yugabyte-simple-c-app
+```
+
+## Provide connection parameters
+
+The application needs to establish a connection to the YugabyteDB cluster. To do this:
+
+1. Open the `sample-app.c` file.
+
+2. Set the following configuration-related macros:
+
+    - **HOST** - the host name of your YugabyteDB cluster. To obtain a Yugabyte Cloud cluster host name, sign in to Yugabyte Cloud, select your cluster on the **Clusters** page, and click **Settings**. The host is displayed under **Network Access**.
+    - **PORT** - the port number that will be used by the driver (the default YugabyteDB YSQL port is 5433).
+    - **DB_NAME** - the name of the database you are connecting to (the default database is named `yugabyte`).
+    - **USER** and **PASSWORD** - the username and password for the YugabyteDB database. If you are using the credentials you created when deploying a cluster in Yugabyte Cloud, these can be found in the credentials file you downloaded.
+    - **SSL_MODE** - the SSL mode to use. Yugabyte Cloud [requires SSL connections](../../../cloud-secure-clusters/cloud-authentication/#ssl-modes-in-ysql); use `verify-full`.
+    - **SSL_ROOT_CERT** - the full path to the Yugabyte Cloud cluster CA certificate.
+
+3. Save the file.
+
+## Build and run the application
+
+Build the application with gcc or clang:
```

Comment:
How would you build with clang?

Reply:
This mimics our other docs, saving that one for another day